browser/css/jsdialogs.css

```diff
@@ -1448,3 +1448,11 @@ input[type='number']:hover::-webkit-outer-spin-button {
 #FormulaDialog #box3 {
 	display: grid;
 }
+
+#TemplateDialog8 {
```

Comment:
are you sure this '8' isn't something automatically added? (we add numbers to avoid duplication, maybe will be better to use "starts with TemplateDialog" ?